.github/workflows/pulls.yml

```diff
@@ -0,0 +1,88 @@
+name: Test All Grammar
+
+on:
+  pull_request:
+    branches:
+      - "main"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    outputs:
+      self_mutation_happened: ${{ steps.self_mutation.outputs.self_mutation_happened }}
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
+        with:
+          node-version-file: '.nvmrc'
+      - name: Install dependencies
+        run: npm ci
+      - name: build
+        run: npm run build
+      - name: Setup Copywrite tool
```

Comment:
Can we use a separate job for the headers? So it's in line with all other repositories?
https://github.com/hashicorp/terraform-ls/blob/main/.github/workflows/add-copyright-headers.yml

Reply:
While we could do that, I don't think it's worth it. The copyright headers part is only four simple lines that set it up and then run the tool. I don't think it's worth duplicating that just for having a different git author on them. But I don't feel strongly about this, so happy to change it, if you feel strongly about it.

Reply:
Put in a separate PR adding that workflow: https://github.com/hashicorp/syntax/pull/105
I'll update this one to stop caring about copyright headers 👍